packages/pwa-kit-build/src/configs/webpack/config.js

```diff
@@ -244,7 +254,11 @@ const clientOptional = baseConfig('web')
                 ...optional('worker', './worker/main.js'),
                 ...optional('core-polyfill', resolve(projectDir, 'node_modules', 'core-js')),
                 ...optional('fetch-polyfill', resolve(projectDir, 'node_modules', 'whatwg-fetch'))
-            }
+            },
+            plugins: [
+                ...config.plugins,
+                analyzeBundle && getBundleAnalyzerPlugin('client-optional')
```

Comment:
Note:  we name this bundle `pwa-others`, probably need to change it to something more relavant to `retail-react-app` taking into the account we have changed it to template-retail-react-app

Reply:
- Let's have both the webpack name and the name of analyzer reports to be the **same name**.
- As for which name to use, I think it should be **general** because this webpack config is meant to be used for other templates too (not just the retail react app). Instead of `pwa-others` my idea is to use `client-optional`. We've already name the previous bundle 'client' and this is going to be its optional bundle.

Reply:
agreed on that, I've changed the bundle name to `client-optional` for consistency